From ba3a6c1da5c4b201efb62b8e24efc928a0c68919 Mon Sep 17 00:00:00 2001
From: Shawn Anastasio <shawnanastasio@yahoo.com>
Date: Mon, 13 Aug 2018 23:55:54 -0500
Subject: [PATCH 1/3] DONTMERGE third_party/node: Use system nodejs binary

Don't merge this upstream. Should eventually be replaced with
logic to determine if a suitable system node binary is present
and fall back to included ones if not.
---
 third_party/node/node.py | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/third_party/node/node.py b/third_party/node/node.py
index 8097e2c49acf..10e6a16eab12 100755
--- a/third_party/node/node.py
+++ b/third_party/node/node.py
@@ -10,11 +10,12 @@ import sys
 
 
 def GetBinaryPath():
-  return os_path.join(os_path.dirname(__file__), *{
-    'Darwin': ('mac', 'node-darwin-x64', 'bin', 'node'),
-    'Linux': ('linux', 'node-linux-x64', 'bin', 'node'),
-    'Windows': ('win', 'node.exe'),
-  }[platform.system()])
+  return "/usr/bin/node"
+  #return os_path.join(os_path.dirname(__file__), *{
+  #  'Darwin': ('mac', 'node-darwin-x64', 'bin', 'node'),
+  #  'Linux': ('linux', 'node-linux-x64', 'bin', 'node'),
+  #  'Windows': ('win', 'node.exe'),
+  #}[platform.system()])
 
 
 def RunNode(cmd_parts, stdout=None):
-- 
2.17.1

